src/driver.toit

```diff
@@ -189,102 +81,101 @@ class Scmd:
     reg_.write_u8 DRIVER_ENABLE_ 0x01
 
   disable:
-    // disable driver functions for H bridges
     reg_.write_u8 DRIVER_ENABLE_ 0x00
 
-  reset_:
-    // reset driver functions for H bridges
-
-  set_drive motorNum/int direction/int level/int:
-    // drive a motor at given level
-    // param motorNum/int: motor number from 0 to 33
-    // param level/int: level from 0 to 255
-    // param direction/int: direction from 0 (forward) or 1(backward)
-    // no return value
-
-    level = round_up (level + 1 - direction/2) 10.to_int
```

Comment:
I didn't fully understand this statement.
The original C code didn't have it either, so I removed it.